src/preprocess.py

```diff
@@ -178,6 +179,47 @@ def construct_documents_mapping(collection: OrdDict[str, Any]) -> OrdDict[int, s
         mapping[idx] = key
     return mapping
 
+@dataclass
+class StatCollection:
+    """
+    Usefull stats of a collection:
+
+    'self.nb_docs' is the number of docs in the collection
+    'self.doc_stats' is a dictionary that provides for a given article id its collection
+    """
+    nb_docs: int
+    doc_stats: OrdDict[int, OrdDict[str, float]]
+
+def get_stats_document(document: List[str]) -> OrdDict[str, float]:
+    """Computes usefull stats for a document
+    
+    Arguments:
+        document {List[str]} -- lemmatized document
+    
+    Returns:
+        Dict[str, float] -- max frequency, mid frequency and unique count in the document
+    """
+    stats={}
+    frequencies = Counter(document)
+    stats["freq_max"] = max(frequencies.values())
+    stats["moy_freq"] = sum(frequencies.values())/len(frequencies)
+    stats["unique"] = len(frequencies.values())
+    return stats
+
+def get_stats_collection(processed_collection: OrdDict[int, List[str]]) -> StatCollection:
+    """Computes usefull stats for the collection
+    
+    Arguments:
+        processed_collection {Dict[str, List[str]]} -- collection of lemmatized articles
+    
+    Returns:
+        StatCollection -- Statictics of the collection
+    """
+    stats={}
```

Comment:
![image](https://user-images.githubusercontent.com/1516762/79465994-4ab3b980-7ffc-11ea-998f-c15cc6a96c09.png)
Should be `OrderedDict()` instead of `{}`

Reply:
done